proto/kuksabase.proto

```diff
@@ -0,0 +1,128 @@
+// Copyright Robert Bosch GmbH, 2022. Part of the Eclipse Kuksa Project.
+//
+// All rights reserved. This configuration file is provided to you under the
+// terms and conditions of the Eclipse Distribution License v1.0 which
+// accompanies this distribution, and is available at
+// http://www.eclipse.org/org/documents/edl-v10.php
+
+//This is a base proto file for databroker and kuksa-val-basic
+//function set
+
+
+syntax = "proto3";
+ 
+//I added V1 as in databroker. Is this good practice?
+package kuksa.val.v1;
+import "google/protobuf/timestamp.proto";
+
+import "kuksatypes.proto";
+
+
+
+option go_package = "/kuksa_grpc_proto";
+
+/*
+Not on autorization: We assume to send auth-token or auth-uuid in metadata
+Token is a JWT like the examples here https://github.com/eclipse/kuksa.val/tree/master/kuksa_certificates/jwt
+See also https://github.com/eclipse/kuksa.val/blob/master/doc/jwt.md
+Upon reception of auth-token, server shall generate an auth_id in metadata
+that the client can use instead of auth_token in subsequent calls
+*/
+
+// The connecting service definition.
+service kuksa_val {
+  // Returns a list of requested data points.
+  rpc get (GetRequest) returns (GetResponse) {}
+  rpc set (SetRequest) returns (SetResponse) {}
```

Comment:
This means the feeders call the one RPC per datapoint and not use a streaming call. Right?

Reply:
Yes, see my comment in the original post. 
Not sure we should integrate streaming (it seems to make sense for some high-frequency, possibly single datapoint sources), or not (none of our open stuff uses it, and GRPC is pretty efficient anyway, does it make a difference)

Reply:
I vote: Lets keep this simple first. Set is setting ONE dp. We can always come back and optimize